docs/develop/datatypes.md

```diff
@@ -70,7 +73,7 @@ To learn the API methods, check the [api reference for set collection data types
 ## FAQs
 <details>
   <summary>If I perform an API call to get a subset of data from a Dictionary item, will the size of the entire item count toward the per GB transfer costs?</summary>
-No, it will not. For example, if you perform the API call DictionaryGetField to get one 5-kilobyte field in a dictionary where the entire dictionary item is 50 kilobytes, only 5 kilobytes count towards my per GB transfer costs.
+No, it will not. For example, if you perform the API call DictionaryGetField to get one 5-kilobyte field:value pair from a dictionary where the entire dictionary item is 50 kilobytes, only 5 kilobytes count towards my per GB transfer costs.
```

Comment:
Nit: `towards my per GB transfer cost` seems weird using `my` here? Should it be `your`?

Reply:
Good catch. fixed!